service-loadbalancer/index-template.http

```diff
@@ -0,0 +1 @@
+404 page not found
```

Comment:
curious, why do you need this? doesn't haproxy already have something like:
```
$ curl http://haproxy/asdfsadf
<html><body><h1>503 Service Unavailable</h1>
No server is available to handle this request.
</body></html>
```

I like an explicit 404 page, just asking if you have some other concrete case.